exporters/etw/include/opentelemetry/exporters/etw/etw_random_id_generator.h

```diff
@@ -17,6 +17,8 @@ namespace trace
 
 class ETWRandomIdGenerator : public IdGenerator
 {
+public:
+  ETWRandomIdGenerator() : IdGenerator(true) {}
```

Comment:
I think GUID is not a truly random value. We should keep it false?

Reply:
Good question.

The true flag means: "at least the right-most 7 bytes of the trace ID MUST be random (or pseudo-random)".

I don't know the internal GUID format to decide which byte goes where, and beside, there are many different GUID formats.

We can conservatively put false here, it is legal even if the data actually is random.

Reply:
Changed to false.